docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -198,43 +198,192 @@ This table can then be queried in Trino::
 
     SELECT * FROM cassandra.mykeyspace.users;
 
-Data types
-----------
-
-The data types mappings are as follows:
-
-================  ======
-Cassandra         Trino
-================  ======
-ASCII             VARCHAR
-BIGINT            BIGINT
-BLOB              VARBINARY
-BOOLEAN           BOOLEAN
-DATE              DATE
-DECIMAL           DOUBLE
-DOUBLE            DOUBLE
-FLOAT             REAL
-INET              IPADDRESS
-INT               INTEGER
-LIST<?>           VARCHAR
-MAP<?, ?>         VARCHAR
-SET<?>            VARCHAR
-SMALLINT          SMALLINT
-TEXT              VARCHAR
-TIMESTAMP         TIMESTAMP(3) WITH TIME ZONE
-TIMEUUID          UUID
-TINYINT           TINYINT
-TUPLE             ROW with anonymous fields
-UUID              UUID
-UDT               ROW with field names
-VARCHAR           VARCHAR
-VARINT            VARCHAR
-================  ======
-
-Any collection (LIST/MAP/SET) can be designated as FROZEN, and the value is
-mapped to VARCHAR. Additionally, blobs have the limitation that they cannot be empty.
-
-Types not mentioned in the table above are not supported.
+.. _cassandra-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and Cassandra each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Cassandra type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Cassandra types to the corresponding Trino types according to
+the following table:
+
+.. list-table:: Cassandra type to Trino type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - Cassandra database type
+    - Trino type
+    - Notes
+  * - ``ASCII``
+    - ``VARCHAR``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``BLOB``
+    - ``VARBINARY``
+    - Blobs have the limitation that they cannot be empty.
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``DECIMAL``
+    - ``DOUBLE``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``FLOAT``
+    - ``REAL``
+    -
+  * - ``INET``
+    - ``IPADDRESS``
+    -
+  * - ``INT``
+    - ``INTEGER``
+    -
+  * - ``LIST<?>``
+    - ``VARCHAR``
+    - Any collection (``LIST``, ``MAP``, or ``SET``) can be designated as
+      `FROZEN <https://docs.datastax.com/en/dse/6.8/cql/cql/cql_using/refCollectionType.html>`_.
```

Comment:
External links should only go to official resources. In this case, if needed, Cassandra documentation https://cassandra.apache.org/doc/latest/cassandra/cql/types.html#frozen

Reply:
This C* link to frozen doesn't seem to work. Will remove link.